gladier_tools/tests/posix/test_untar.py

```diff
@@ -0,0 +1,32 @@
+from unittest.mock import Mock
+import tarfile, os
+import pytest
+from gladier_tools.posix.untar import untar_file
+
+
+def isfile_sub(data):
+        return False
+
+def test_no_file(monkeypatch):
+    monkeypatch.setattr(os.path, 'isfile', isfile_sub)
```

Comment:
It's good practice to re-use patching code used in other modules. This could be added as a fixture using:

```
@pytest.fixturue
def mock_isfile(monkeypatch):
    """Mock isfile. Returns False by default"""
    monkeypatch.setattr(os.path, 'isfile', Mock(return_value=False))
    return os.path.isfile
```

Since the return value of `isfile` needs to be different, you can change it according to your needs:

```
def test_mkdir(monkeypatch, mock_isfile):
    mock_isfile.return_value = True
    ...
```